### ISSUE TITLE
index.graphql and any schema files named index

### ISSUE THREAD

**Issue by sonyseng** [Urigo/merge-graphql-schemas#135]
I just started using this library and this stumped me for a short while. Just wanted to point out that the code filtering out `index.js` and `index.ts` is a bit more general than what the README states and will filter out any file named `index`. It just so happened that my graphql schema files were named `index.graphql`. 

file_loader.js

```javascript
  const files = schemafiles
        .map(f => ({ f, pathObj: path.parse(f) }))
        .filter(({ pathObj }) => pathObj.name.toLowerCase() !== 'index')
        .filter(({ pathObj }) => extensions.includes(pathObj.ext))
```

**Comment by cfnelson** [comment]
@sonyseng Would be happy to accept a PR if you would like to contribute. 🙌  You could update the documentation to make this behaviour more clear to anyone using the pkg. 
We potentially could also accept a PR if given some thought as well to change the behaviour you described above.

**Comment by kamilkisiela** [comment]
https://github.com/ardatan/graphql-toolkit/pull/188

**Comment by kamilkisiela** [comment]
Fixed with `v1.7.0`
